settings.gradle

```diff
@@ -1 +1,3 @@
-include ':AndroidStealth', ':libraries:FileChooser', ':libraries:droidmorphing-lib', ':libraries:DirectoryChooser'
+include ':AndroidStealth', ':libraries:FileChooser', ':libraries:DirectoryChooser', ':droidmorphing-lib'
+project(':droidmorphing-lib').projectDir = new File('libraries/droid-morph/droidmorphing-lib')
```

Comment:
Just to make sure I get it right? This just includes it as files that we can import? And is built as a part of the whole project? Or is it included in the compile some other way?